deployments/dev/environment.py

```diff
@@ -45,7 +45,7 @@ def env() -> Mapping[str, Optional[str]]:
 
         'AZUL_TDR_SOURCE': 'tdr:broad-jade-dev-data:snapshot/hca_dev_20200817_dssPrimaryOnly',
         'AZUL_TDR_DCP2EBI_SOURCE': 'tdr:broad-jade-dev-data:snapshot/hca_dev_20200828_ebi17src___20200828',
-        'AZUL_TDR_IT2EBI_SOURCE': 'tdr:broad-jade-dev-data:snapshot/hca_dev_ebi___20200819',
+        'AZUL_TDR_IT2EBI_SOURCE': 'tdr:broad-jade-dev-data:snapshot/hca_dev_20200828_ebi17src___20200828',
```

Comment:
Unless we have a reason not to, we should always use the same source for the IT catalog as the corresponding non-IT catalog.